tests/Tests/500__jupyterhub/test-minimal-image.robot

```diff
@@ -35,6 +35,21 @@ Can Spawn Notebook
     Fix Spawner Status
     Spawn Notebook With Arguments    image=s2i-minimal-notebook
 
+Verify Tensorflow Can Be Installed In The Minimal Python Image Via Pip
+    [Documentation]    Verify Tensorflow Can Be Installed In The Minimal Python image via pip
+    [Tags]    ODS-555
+    Open With JupyterLab Menu    File    New    Notebook
+    Sleep    1
+    Maybe Close Popup
+    Close Other JupyterLab Tabs
+    Maybe Close Popup
+    Sleep    1
+    Add and Run JupyterLab Code Cell In Active Notebook    !pip install tensorflow
+    ${output} =    Run Cell And Get Output
+    ...    !pip show tensorflow | grep Name: | awk '{split($0,a); print a[2]}' | awk '{split($0,b); printf "%s", b[1]}'
+    Should Be Equal    tensorflow    ${output}
```

Comment:
not sure if this check is actually needed.

Btw, maybe an additional check it might be worth to do is importing tensorflow in a new cell and check that no errors are returned @lugi0

Reply:
I think this check at least shows that tensorflow has been installed and can be seen by pip, which is almost the same as being able to import it (can't think of a situation where pip sees it but you cannot use it). 
I know @pranavsuryaw was updating this PR to also include other steps, so maybe he could also add the import check

Reply:
hmm I can't find a similar situation too, but I thought it was worth. But, it's okay if you feel line 50 is enought it's okay :)